### ISSUE TITLE
Hello Sir : Could you add Save to Cloud Storage

### ISSUE THREAD

**Issue by Tri44** [jameszah/ESP32-CAM-Video-Telegram#22]
Could you add save to cloud storage like Firebase for example?
after it record to Spiffs then upload it to Firebase Storage using one of this library
https://github.com/mobizt/Firebase-ESP-Client

Thanks you

**Comment by jameszah** [comment]
Good idea - i nearly completed that when I was called away.  Starting again.
